packages/firestore/src/util/proto_byte_string.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PlatformSupport } from '../platform/platform';
+
+/**
+ * Immutable class that represents a "proto" byte string.
+ *
+ * Proto byte strings can either be base64 strings or Uint8Arrays when sent
+ * on the wire, but they are always typed to string. This class abstracts away
```

Comment:
I think we can remove ", but they are always typed to string". This seems to not play a major role in this class.